### ISSUE TITLE
Dress Rehearsal Grading

### ISSUE THREAD

**Issue by Arushi-maker** [KinetekEnergy/ByteBuffoons-Pages#26]
USAGE OF:
- [ ] Frontend
- [ ] Backend
- [ ] API
- [ ] Agile

1. HOOK: 
Points: 3.6-4.0
Reason:

2. KNOWLEDGE:
Points: 3.6-4.0
Reason:

3. VALUE:
Points: 0.6-1.0
Reason:

4. WOW FACTOR:
Reason:

**Comment by JasonGao76** [comment]
Criteria/Key Features:
- [X] Frontend
- [X] Backend
- [X] API
- [X] Agile (could have a little more though)

HOOK:
Points: 4.0
Reason: Started off with the beautiful home screen with the images of the cookies and introductory screens. They welcomed us with enthusiasm and gave us hands on experience with the project.

KNOWLEDGE:
Points: 4.0
Reason: Explained how the data table and API handled and sent data to the frontend and why we can see what we see. Explained how the storage and various features worked as well. Explained how script and getting images worked too and is very interesting.

VALUE:
Points: 1.0
Reason: Who doesn't like cookies? Being able to see the different types of cookies and implementing an order feature is something that is very unique and has potential to grow into something like a small business!

WOW FACTOR: 0.8
Reason: Project was amazing and the explanations of all the various complex functionality were very impressive and interesting. I would give it 1.0 but I think I need to dock a tiny bit because the Agile wasn't really talked about, so maybe you guys can give it a few more sentences of explanation. Otherwise, this project is amazing with complex functionality!

Total: 4 + 4 + 1 + 0.8 = 9.8/10

**Comment by will-w-cheng** [comment]
- [x] Frontend
- [x] Backend
- [x] API
- [x] Agile 


HOOK:
Points: 3.6-4.0 
Reason:
While there was a lot of good stuff with the cookie presentation I wish that there was more of a demo aspect to the stuff.

KNOWLEDGE:
Points: 3.6-4.0
Reason:
Shared the cookie knowledge and how there was a request and web scraping that clearly exemplifies their knowledge. Really good job here.

VALUE:
Points: 0.6-1.0
Reason:
Really good basis for a lot of real shopping and work for the value and features such as adding cookies for billing prices 

WOW FACTOR:
Reason:
Visual appeasement was pretty impressive, the cookies cookies in the catalog was done really well and I personally was wowed from all the functionality there was such as the different billing options.

9.8 / 10

**Comment by Delta760** [comment]
> USAGE OF:
> 
> * [ x]  Frontend
> * [ ]  Backend
> * [ ]  API
> * [ ]  Agile
> 
> 1. HOOK:
>    Points: 3.9
>    Reason: Very good
> 2. KNOWLEDGE:
>    Points: 3.6-4.0
>    Reason: Was very clear and precise
> 3. VALUE:
>    Points: 1
>    Reason: I am severely addicted to cookies
> 4. WOW FACTOR:
>    Reason: A lot of advanced code was used effectively

8.9/9

**Comment by eunseolim123** [comment]
USAGE OF:

- [ ]  Frontend

- [ ]  Backend

- [ ]  API

- [ ]  Agile

HOOK: 3.6-4.0
Points: 3.7
Reason: hook could be more interesting

KNOWLEDGE: 3.6-4.0
Points: 4.0
Reason: detailed explanation, knowledgeable on code

VALUE: 0.6-1.0
Points: 1.0
Reason: useful, visually appealing, frontend and backend connected well

WOW FACTOR:
Reason: visuals, frontend was very solid

**Comment by jm1021** [comment]
Criteria/Key Features:


- [ ] Frontend
- [ ]  Backend
- [ ]  API
- [ ]  Agile

HOOK:
Points: 3.6-4.0
Reason:
Started with design.  Shared how their design led to implementation.   

KNOWLEDGE:
Points: 3.6-4.0
Reason:
Talked about CSS Grid.   Had a Postman process for adding a cookie.

VALUE:
Points: 0.6-1.0
Reason:
Very much like a cookie shopping

WOW FACTOR:
Reason:
Sharing a lot on process and workflow.  


9.4/10

**Comment by AnanyaAsudani** [comment]
HOOK:
Points: 3.8
Reason: Began the presentation with a hook and got the audience interested in the project. The hook could be a little more elaborate

KNOWLEDGE:
Points: 3.9
Reason: Well done and well thought out. The group thoroughly explains the project and the code used for it. Explains both backend and frontend

VALUE:
Points: 0.7
Reason: It is well thought out but not a largely impactful topic. The project allows viewers to buy cookies.

WOW FACTOR: 0.9
Reason: Agile was glossed over but the frontend was well done and the layout was nice.

Total: 3.8 + 3.9 + 0.7 + 0.9 = 9.3/10

**Comment by Arushi-maker** [comment]
Self-Grading

USAGE OF:

- [X] -  Frontend
- [X] -  Backend
- [X] -  API
- [X] -  Agile

HOOK:
Points: 3.7
Reason: Began with the beginning process and agile methodology. Moved on to discussing project proposals and the implementation of Figma. Allowed people to engage with the website as we began discussing. Could contain more professionalism and more of a demo aspect. 

KNOWLEDGE:
Points: 4.0
Reason: Begins with the 3 card grid for the cookies trying to replicate the Figma design for the catalog. Moreover, began to discuss the cart features and checkout features. The API stores the data through AWS and we were able to generate 25 different cookies, while only having data for 8. Additionally, we used scraper to pull images from a recipe site for cookies so that we don't have to do it manually. The checkout screen also features a moving background with cookies to enforce the cookie theme of the website. We are working towards implementing a "added to cart" sign to pop up once add to cart button is clicked for a cookie.

VALUE:
Points: 1.0
Reason: Purpose of being a cookie e-commerce website was reached. Efficiently offer variety of cookies, and a checkout service.

WOW FACTOR:
Reason: Visuals were appealing, humorous presentation while also very informative. 

Self-Grade Total: 8.7/9.0
